README.md

```diff
@@ -176,25 +176,30 @@ search for a Python interpreter matching that version in the following order:
+Since uv has no dependency on Python, it can install into virtual environments other than
 its own. For example, setting `VIRTUAL_ENV=/path/to/venv` will cause uv to install into
-`/path/to/venv`, no matter where uv is installed.
-
-uv can also install into arbitrary, even non-virtual environments by providing a `--python` argument
-to `uv pip sync` or `uv pip install`. For example, `uv pip install --python=/path/to/python` will
-install into the environment linked to the `/path/to/python` interpreter.
-
-For convenience, `uv pip install --system` will install into the system Python environment, as an
-approximate shorthand for, e.g., `uv pip install --python=$(which python3)`. Though we generally
-recommend the use of virtual environments for dependency management, `--system` is intended to
-enable the use of uv in continuous integration and containerized environments.
-
-The `--system` flag is also used to opt-in to mutation of system environments. For example, when
-the `--python` argument is used to implicitly request a system interpreter, e.g., if `--python 3.12`
-finds `/usr/lib/python3.12`, then the `--system` flag is required to allow modification of this non-virtual
-Python environment. When the `--system` flag is provided, uv will ignore any interpreters that are not in
-[PEP 405 compliant](https://peps.python.org/pep-0405/#specification) virtual environments. Similarly, if
-the `--system` flag is not provided, uv will ignore any interpreters that are in not virtual environments.
+`/path/to/venv`, regardless of where uv is installed.
+
+uv can also install into arbitrary, even non-virtual environments, with the `--python` argument
+provided to `uv pip sync` or `uv pip install`. For example, `uv pip install --python=/path/to/python`
+will install into the environment linked to the `/path/to/python` interpreter, and
+`uv pip install --python=$(which python)` will install into the environment linked to the
+current interpreter on the `PATH`. Another way to install into the environment linked to the
+current interpeter is to invoke `uv` as a module (e.g., `python -m uv pip install`).
+
+For convenience, `uv pip install --system` will install into the system Python environment.
+Although we generally recommend using virtual environments for dependency management,
+`--system` is appropriate in continuous integration and containerized environments.
+
+The `--system` flag is also used to opt in to mutating system environments. For example, when
+the `--python` argument is used to implicitly request a system interpreter (e.g., if `--python 3.12`
+finds `/usr/lib/python3.12`), then the `--system` flag is required to allow modification of this non-virtual
+Python environment.
+
+Unless the `--python` argument is provided, or uv is invoked as a module,
+uv will ignore any interpreters not in
+[PEP 405 compliant](https://peps.python.org/pep-0405/#specification) virtual environments,
+whether or not the `--system` flag is provided.
```

Comment:
I wasn't sure how to interpret this:
> When the `--system` flag is provided, uv will ignore any interpreters that are not in [PEP 405 compliant](https://peps.python.org/pep-0405/#specification) virtual environments. Similarly, if the `--system` flag is not provided, uv will ignore any interpreters that are in not virtual environments.

This paragraph is a guess, but it may not be correct.

Reply:
OK, since I didn't understand your version, I figured it must have included a misstatement :)
My text was an attempt to rephrase yours under the assumption it was correct as written, which means mine is also wrong. I'll revert and let you patch your version.

Reply:
1c8c26b7